api/SVGPoint.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "SVGPoint": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGPoint",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": false,
```

Comment:
Hmm, I think this should be true? SVGPoint was part of the SVG spec and hangs around in [some dark corners](https://www.w3.org/TR/geometry-1/#intro)

Reply:
You're absolutely right, not sure why I put non-standard + deprecated, since I even linked to the (latest) SVG specification, which includes it.

Anyways, I have fixed that now in 9455f7d.